src/ts/components/pages/RegisterForm/RegisterForm.tsx

```diff
@@ -86,10 +79,12 @@ export class RegisterForm extends React.PureComponent<{}, RegisterFormState>{
                             onChange={event => this.setState({ password: event.target.value, })} />
                         {this.renderSelect()}
                     </div>
+                    {/* Password */}
                     <div className="section">
                         <input type="password" className="leftInput" placeholder={RegisterFormLabels.password} required onChange={event => this.setState({ password: event.target.value, })} />
```

Comment:
As this is refactoring this could easily be made into multiple lines for readability